py3dtilers/Common/tiler.py

```diff
@@ -257,7 +262,10 @@ def create_tileset_from_groups(self, groups: Groups, extension_name=None):
         geometric_errors = self.args.geometric_error if hasattr(self.args, 'geometric_error') else [None, None, None]
         with_normals = False if self.args.no_normals else True
 
-        tree = LodTree(groups, self.args.lod1, create_loa, self.args.with_texture, geometric_errors, self.args.texture_lods)
+        if self.args.as_lods:
+            tree = LodTree.vertical_hierarchy(groups, geometric_errors)
+        else:
+            tree = LodTree(groups, self.args.lod1, create_loa, self.args.with_texture, geometric_errors, self.args.texture_lods)
```

Comment:
To avoid make things clearer here, passing a 'config' parameter to the lod tree could clarify the code. It would also to do this "if" in the Lod_tree file, which belong there